### ISSUE TITLE
Can't login to Evernote

### ISSUE THREAD

**Issue by someone1233** [baumgarr/Nixnote2#511]
Hi, I've used Nixnote2 for a while & it's been working great! Recently I became logged out of Evernote somehow (maybe an update?) and received a popup window to log back in. I typed in my username and clicked the green "Continue" button but the page does not seem to advance past this point. This was a few days ago, I tried again now & the problem persists. Can you please check? Thank you!!

![Screenshot from 2021-07-29 13-28-13](https://user-images.githubusercontent.com/32484427/127537892-7b9f9692-65fa-44cb-a0e1-231b4431dd5a.png)

**Comment by robert7** [comment]
Hi,
sorry I don't have any time to support this project anymore, but existing version should work OK. I verified just now with Ubuntu 20.04 that the login still works OK.

- first check whenewer are you using the newest 2.1 version or the older 2.0; switch to the new (you are reporting the problem in the old repository for 2.0)
- try reinstall and/or use new data directory (configDir commandline option) ... see steps in https://github.com/robert7/nixnote2/wiki/HowTo---What-to-do-if-the-app-crashes
- if you are using 2FA (you should), then you can try to disable it for short time, to verify, if this is the problem

anyway, I expect the problem to be related to your installation

**Comment by someone1233** [comment]
Ok, I will check all these things & see if they resolve the issue. Thank you for taking the time to give advice even though you're not supporting the project anymore.

**Comment by someone1233** [comment]
Hello again, I wanted to see if you maybe had any other suggestion I could try -- so far no luck getting it to work again. I verified the app is at the latest version (2.1.6) and also uninstalled/reinstalled via the software manager. I tried setting up a new account via "user account maintenance" and also changed the configDir with the command line. Every time it's the same problem -- the login window appears, I can type my username/email address, but I can't seem to interact with the green "continue" button in order to advance the page. It doesn't respond to either mouse click or enter keypress. I don't know why it won't let me click this button but there doesn't seem to be any way around it. Any thought? If not no worries, there's always the web interface. thank you for your time!

**Comment by robert7** [comment]
You also tried the AppImage version? (it could be problem of dependencies on your workstation, this could then work using AppImage version, as this has dependencies included in the binary)

Further what to try: try to login on different computer (or e.g. via Virtualbox VM on your computer)

**Comment by someone1233** [comment]
tyvm!! I downloaded the appimage & ran it once, for whatever reason I was able to interact with the login button now! It took my login info & prompted to authorize Nixnote's access to the account for 1 year (I guess the previous authorization expired and that's why it got logged out to begin with), then I closed it and opened the regular version on my system and all is good. hurray! so happy to have Nixnote back, the web interface is sooo pop-up riddled. thanks a bunch! xoxo
